docs/enterprise/thales-install-guide.md

```diff
@@ -22,45 +22,45 @@ have the following files supplied by Chain:
   
 You must also perform the following steps:
 
--  Set up the HSM(s) and the client host(s) according to the
+1. Set up the HSM(s) and the client host(s) according to the
    directions from Thales
-   - Install CodeSafe software on client host(s)
-   - Initialize the Remote File System
-   - Configure host(s) and HSM(s) with each other’s IP addresses
-   - Create the “Security World” with the Secure Execution Engine
+   1.1 Install CodeSafe software on client host(s)
```

Comment:
Check the "View" button on the top of this file to see it on github. (I'm not sure github's markdown renderer is 100% the same as whatever we use for chain.com, but it should be close. Also I am pretty sure "1.1" isn't considered a list in markdown; see https://daringfireball.net/projects/markdown/syntax.)